src/components/Navigator/NavigatorCard.vue

```diff
@@ -24,7 +24,13 @@
           </Reference>
         </div>
         <slot name="post-head" />
-        <div class="card-body">
+        <div
+          class="card-body"
+          @keydown.meta.up.capture.prevent="focusFirst"
+          @keydown.meta.down.capture.prevent="focusLast"
```

Comment:
Shouldn't we be using `alt` and not `cmd` for first/last ?

Reply:
https://github.com/apple/swift-docc-render/pull/79/commits/1f9ba274e05fe857da47b80148eef7f8ac1d036a

Reply:
I think you did not understand me. I think it should be alt + up, not ctrl/cmd.